.github/workflows/websocket__build-host-tests.yml

```
@@ -0,0 +1,74 @@
+name: "websocket: build/host-tests"
+
+on:
+  push:
+  #  branches:
+  #    - master
+  pull_request:
+    types: [opened, synchronize, reopened, labeled]
+jobs:
+  host_test_websocket:
+    if: contains(github.event.pull_request.labels.*.name, 'websocket') || github.event_name == 'push'
+    name: Host Tests
+    runs-on: ubuntu-20.04
+    permissions:
+        contents: write
+    container: espressif/idf:latest
+    env:
+      COMP_DIR: esp-protocols/components/esp_websocket_client
+    steps:
+      - name: Checkout esp-protocols
+        uses: actions/checkout@v3
+        with:
+          path: esp-protocols
+      - name: Build and Test
+        shell: bash
+        run: |
+          . ${IDF_PATH}/export.sh
+          cd $GITHUB_WORKSPACE/${{ env.COMP_DIR }}/test/host_test
+          idf.py build
+          ./build/websocket-host.elf -r junit -o junit.xml
```

This is not a test, why the `junit` parameters?
```suggestion
          ./build/websocket-host.elf
```